docs/how-to/cos-tls.tf

```diff
@@ -0,0 +1,17 @@
+# Note: The deployment order matters since the 'traefik:certificates' integration depends on 'module.ssc'
+#   'terraform apply -target module.ssc'
+#   'terraform apply'
+
+module "ssc" {
+  source = "git::https://github.com/canonical/self-signed-certificates-operator//terraform"
+  model  = "external-ca"
+}
+
+module "cos" {
+  source                          = "git::https://github.com/canonical/observability-stack//terraform/cos"
+  model                           = "cos"
+  channel                         = "1/stable"
+  traefik_channel                 = "latest/edge"
+  internal_tls                    = true  # Set to 'false' to disable inter-model TLS
```

Comment:
This should be "intra-model":
- intra-model: inside the model
- inter-model: between models